loaders/src/glTF/2.0/glTFLoaderExtension.ts

```diff
@@ -107,19 +109,21 @@ export interface IGLTFLoaderExtension extends IGLTFBaseLoaderExtension, IDisposa
      * @param context The context when loading the asset
      * @param textureInfo The glTF texture info property
      * @param assign A function called synchronously after parsing the glTF properties
+     * @param textureDataType type of data held by the texture
      * @returns A promise that resolves with the loaded Babylon texture when the load is complete or null if not handled
      */
-    loadTextureInfoAsync?(context: string, textureInfo: ITextureInfo, assign: (babylonTexture: BaseTexture) => void): Nullable<Promise<BaseTexture>>;
+    loadTextureInfoAsync?(context: string, textureInfo: ITextureInfo, assign: (babylonTexture: BaseTexture) => void, textureDataType: TextureDataType): Nullable<Promise<BaseTexture>>;

```

Comment:
It think this is a breaking change? Existing glTF loader extensions will not compile right? We should discuss this.

Reply:
I think it's not.

If I do this in KHR_texture_transform (just for testing purpose):
```typescript
public loadTextureInfoAsync(context: string, textureInfo: ITextureInfo,
  assign: (babylonTexture: BaseTexture) => void/*, isColorData = true*/): Nullable<Promise<BaseTexture>> {
    [...]
```
it does work.

Reply:
I'm not sure adding a parameter is the right fix for this. It might be better to add a property to the ITextureInfo class instead.